config/locales/mailings.en.yml

```diff
@@ -14,7 +14,7 @@ en:
         account_activation_instructions: Activate your account by clicking the button below
         account_activation_link: Activate your account for our member system by going to %{url}.
         activate_account: Activate account
-        activity_updates_html: 'Do you always want to stay informed about these activities? Follow our %{instagram_page_link_start} Sticky-members Instagram %{link_end} and follow our %{linkedin_page_link_start} our page %{link_end}! Aside from that, you can find all information that you could ever want on our website: %{sticky_site_link_start} svsticky.nl %{link_end}. You can also join our %{whatsapp_promo_link_start} WhatsApp promotion channel %{link_end} to be notified about upcoming activities!'
+        activity_updates_html: 'Do you always want to stay informed about these activities? Follow us on %{instagram_page_link_start} the Sticky-members Instagram %{link_end} and on %{linkedin_page_link_start} LinkedId %{link_end}! Aside from that, you can find all information that you could ever want on our website: %{sticky_site_link_start} svsticky.nl %{link_end}. You can also join our <a href="https://svsticky.nl/promochannel"> WhatsApp promotion channel </a> to be notified about upcoming activities!'
```

Comment:
Have you tested this? Because I am not sure whether you can put plain html in translations as this requires the 'safe html' directive in Ruby/Haml

Reply:
Haven't no. Will change it to use templating instead, then we for sure are safe.